### PR TITLE
K8SPSMDB-961: add `.spec.sharding.balancer.enabled`

### PR DESCRIPTION
**Pull request by pooknull**
[K8SPSMDB-961](https://jira.percona.com/browse/K8SPSMDB-961) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

https://jira.percona.com/browse/K8SPSMDB-961

**DESCRIPTION**
---
**Problem:**
*There is no option to disable balancer for a cluster with `unmanaged: false`.*

**Cause:**
*We don't have option to specify if balancer should be enabled or not.*

**Solution:**
*Add new field `.spec.sharding.balancer.enabled`.*

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?
- [ ] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Are the manifests (crd/bundle) regenerated if needed?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported MongoDB version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/arbiter.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/cross-site-sharded.log) |
| data-at-rest-encryption | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/data-at-rest-encryption.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/data-sharded.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/demand-backup.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/demand-backup-eks-credentials.log) |
| demand-backup-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/demand-backup-physical.log) |
| demand-backup-physical-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/demand-backup-physical-sharded.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/demand-backup-sharded.log) |
| expose-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/expose-sharded.log) |
| ignore-labels-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/ignore-labels-annotations.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/limits.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/liveness.log) |
| mongod-major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/mongod-major-upgrade.log) |
| mongod-major-upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/mongod-major-upgrade-sharded.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/monitoring-2-0.log) |
| multi-cluster-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/multi-cluster-service.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/non-voting.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/one-pod.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/operator-self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/pitr.log) |
| pitr-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/pitr-sharded.log) |
| recover-no-primary | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/recover-no-primary.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/rs-shard-migration.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/scheduled-backup.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/security-context.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/self-healing-chaos.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/service-per-pod.log) |
| serviceless-external-nodes | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/serviceless-external-nodes.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/smart-update.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/storage.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/upgrade.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/upgrade-consistency.log) |
| upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/upgrade-sharded.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/65554659/version-service.log) |
| We run 38 out of 38|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1304/commits/6555465987e9007345d19f8aae82df398e536a45
image: `perconalab/percona-server-mongodb-operator:PR-1304-65554659`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/arbiter.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/cross-site-sharded.log) |
| data-at-rest-encryption | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/data-at-rest-encryption.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/data-sharded.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/demand-backup.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/demand-backup-eks-credentials.log) |
| demand-backup-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/demand-backup-physical.log) |
| demand-backup-physical-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/demand-backup-physical-sharded.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/demand-backup-sharded.log) |
| expose-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/expose-sharded.log) |
| ignore-labels-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/ignore-labels-annotations.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/limits.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/liveness.log) |
| mongod-major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/mongod-major-upgrade.log) |
| mongod-major-upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/mongod-major-upgrade-sharded.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/monitoring-2-0.log) |
| multi-cluster-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/multi-cluster-service.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/non-voting.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/one-pod.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/operator-self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/pitr.log) |
| pitr-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/pitr-sharded.log) |
| recover-no-primary | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/recover-no-primary.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/rs-shard-migration.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/scheduled-backup.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/security-context.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/self-healing-chaos.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/service-per-pod.log) |
| serviceless-external-nodes | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/serviceless-external-nodes.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/smart-update.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/storage.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/upgrade.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/upgrade-consistency.log) |
| upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/upgrade-sharded.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/c1af91f2/version-service.log) |
| We run 38 out of 38|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1304/commits/c1af91f215614307fd9f83a06289bef648e9b138
image: `perconalab/percona-server-mongodb-operator:PR-1304-c1af91f2`

File: pkg/controller/perconaservermongodb/balancer.go [hunk 1]
```diff
@@ -103,10 +103,17 @@ func (r *ReconcilePerconaServerMongoDB) enableBalancerIfNeeded(ctx context.Conte
 	return nil
 }
 
+func (r *ReconcilePerconaServerMongoDB) disableBalancerIfNeeded(ctx context.Context, cr *api.PerconaServerMongoDB) error {
+	if s := cr.Spec.Sharding; !s.Enabled || s.Mongos.Size == 0 || cr.Spec.Unmanaged || s.Balancer.Enabled == nil || *s.Balancer.Enabled {
```
Comment (egegunes) [inline pkg/controller/perconaservermongodb/balancer.go:107]: we shouldn't disable balancer if cluster is unmanaged. if cluster is unmanaged we simply shouldn't touch it.

**Review by egegunes — CHANGES_REQUESTED** [review]

File: pkg/controller/perconaservermongodb/balancer.go (hunk 1 above)
Comment (pooknull) [reply pkg/controller/perconaservermongodb/balancer.go:107]: This if condition is positive when `cr.Spec.Unmanaged == true`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/arbiter.log) |
| balancer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/balancer.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/cross-site-sharded.log) |
| data-at-rest-encryption | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/data-at-rest-encryption.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/data-sharded.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/demand-backup.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/demand-backup-eks-credentials.log) |
| demand-backup-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/demand-backup-physical.log) |
| demand-backup-physical-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/demand-backup-physical-sharded.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/demand-backup-sharded.log) |
| expose-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/expose-sharded.log) |
| ignore-labels-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/ignore-labels-annotations.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/init-deploy.log) |
| finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/finalizer.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/limits.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/liveness.log) |
| mongod-major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/mongod-major-upgrade.log) |
| mongod-major-upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/mongod-major-upgrade-sharded.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/monitoring-2-0.log) |
| multi-cluster-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/multi-cluster-service.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/non-voting.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/one-pod.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/operator-self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/pitr.log) |
| pitr-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/pitr-sharded.log) |
| recover-no-primary | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/recover-no-primary.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/rs-shard-migration.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/scheduled-backup.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/security-context.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/self-healing-chaos.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/service-per-pod.log) |
| serviceless-external-nodes | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/serviceless-external-nodes.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/smart-update.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/storage.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/upgrade.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/upgrade-consistency.log) |
| upgrade-consistency-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/upgrade-consistency-sharded.log) |
| upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/upgrade-sharded.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1304/7ea375e4/version-service.log) |
| We run 41 out of 41|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1304/commits/7ea375e46415e604927f350a9cf18ec3f03d744b
image: `perconalab/percona-server-mongodb-operator:PR-1304-7ea375e4`

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Review by inelpandzic — APPROVED** [review]

**Pull request merged**